.storybook/preview-body.html

```diff
@@ -16,7 +16,7 @@
     }
 
     .be.bcpr {
-        height: 900px;
+        height: 1800px;
```

Comment:
This fixes an issue where the header in ContentPreview can be "pushed" out of bounds. This file is only used for storybook, and the issue does not happen in production.